endpoint/eth.go

```diff
@@ -489,7 +489,7 @@ func (e *Eth) parseRequestFilter(ctx context.Context, filter *request.Filter) (*
 		for i, t := range filter.Topics {
 			f.Topics = append(f.Topics, make([]primitives.Data32, 0))
 			for _, tt := range t {
-				f.Topics[i] = append(f.Topics[i], primitives.Data32FromHex(string(tt)))
+				f.Topics[i] = append(f.Topics[i], tt)
```

Comment:
That makes sense. Just pushed it. Keep in mind `f.Topics` can now be nil, whereas before it would be an empty slice. That shouldn't make a difference, but in case it does, we can apply the assignment only when `filter.Topics` is not itself nil.

Reply:
Yep, agree